lms/djangoapps/lti_provider/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""
```

Comment:
Nit: a docstring here should describe the `lti_provider` package as a whole.  This should be a comment instead.